recipes/hdf5-lz4/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+PPATH="/user/local/hdf5/lib/plugin"
+
+mkdir -vp ${PREFIX}/$PPATH
+
+export CXXLAGS="${CFLAGS}"
+export CPPFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
+
+make -e HDF5_INSTALL="${PREFIX}"
+cp lib* ${PREFIX}/$PPATH
```

Comment:
I suspect I wrote this, but it seems way too blunt, I think the previous line should be `make -e HDF5_INSTALL="${PREFIX}/$PPATH"